tests/v2/validation/provisioning/clustertemplates/README.md

```diff
@@ -0,0 +1,76 @@
+## Cluster template Provisioning Config
+
+For your config, you will need everything in the [Prerequisites](../README.md) section on the previous readme along with and at least one [Cloud Credential](#cloud-credentials). 
+
+Your GO test_package should be set to `provisioning/clustertemplates`.
+Your GO suite should be set to `-run ^TestClusterTemplateRKE1ProvisioningTestSuite$`.
+Please see below for more details for your config. Please see below for more details for your config. Please note that the config can be in either JSON or YAML (all examples are illustrated in YAML).
+
+## Table of Contents
+1. [Getting Started](#Getting-Started)
+
+## Getting Started
+See below a sample config file to run this test:
+```yaml
+rancher:                                   
+  host: ""                                  #required
+  adminToken: ""                            #required
+  clusterName: ""                           
+
+provisioningInput:
+  etcdRKE1:
+    backupConfig:
+        accessKey: <AWS Access Key>
+        secretKey: <AWS Secret Key>
+        endpoint: <S3 endpoint>
+        region: <S3 bucket region>
+        bucketName: <S3 bucket name>
+  RKE1KubernetesVersions: ["v1.30.4-rancher-1"]    #optional. if empty, latest version is considered.
```

Comment:
we're now keeping explicit versions for k8s out of readmes to avoid having to update them each time there's an update.